src/main/scala/djinni/Main.scala

```diff
@@ -325,13 +332,13 @@ object Main {
         .valueName("<true/false>")
         .foreach(x => jniGenerateMain = x)
         .text(
-          "Generate a source file (djinni_jni_main.cpp) that includes the default JNI_OnLoad & JNI_OnUnload implementation from the djinni-support-lib. (default: true)"
+          "Generate a source file (djinni_jni_main.cpp) that includes the default JNI_OnLoad & JNI_OnUnload implementation from the djinni-support-lib. (default: false)"
         )
       opt[Boolean]("jni-generate-main")
         .valueName("<true/false>")
         .foreach(x => jniGenerateMain = x)
         .text(
-          "Generate a source file (djinni_jni_main.cpp) that includes the default JNI_OnLoad & JNI_OnUnload implementation from the djinni-support-lib. (default: true)"
+          "Generate a source file (djinni_jni_main.cpp) that includes the default JNI_OnLoad & JNI_OnUnload implementation from the djinni-support-lib. (default: false)"
```

Comment:
For some odd reason, the whole code block `opt[Boolean]("jni-generate-main")` exists 2 times
That must be due to some accident ...
Can you please remove this one?